lib/utils/screen-manager.js

```diff
@@ -19,6 +19,7 @@ var ScreenManager = module.exports = function (rl) {
   this.extraLinesUnderPrompt = 0;
 
   this.rl = rl;
+  if (rl) rl.activeScreen = this;
```

Comment:
Okay, so if we do this, we need clear relationship. Modifying an object outside your own scope is pretty bad practice and it'll make it hard to debug in the future.